.jenkins/pytorch/short-perf-test-gpu.sh

```diff
@@ -17,14 +17,15 @@ pip install -q awscli --ignore-installed PyYAML
 # Set multipart_threshold to be sufficiently high, so that `aws s3 cp` is not a multipart read
 # More info at https://github.com/aws/aws-cli/issues/2321
 aws configure set default.s3.multipart_threshold 5GB
+UPSTREAM_DEFAULT_BRANCH="$(git remote show https://github.com/pytorch/pytorch.git | awk '/HEAD branch/ {print $NF}')"
 
-if [[ "$COMMIT_SOURCE" == master ]]; then
-    # Get current master commit hash
-    MASTER_COMMIT_ID=$(git log --format="%H" -n 1)
-    export MASTER_COMMIT_ID
+if [[ "$COMMIT_SOURCE" == "$UPSTREAM_DEFAULT_BRANCH" ]]; then
+    # Get current main commit hash
```

Comment:
s/main/default branch/
```suggestion
    # Get current default branch commit hash
```